crates/matrix-sdk-ui/src/timeline/controller/metadata.rs

```diff
@@ -38,6 +38,8 @@ pub(in crate::timeline) struct TimelineMetadata {
     /// This value is constant over the lifetime of the metadata.
     internal_id_prefix: Option<String>,
 
+    /// The `count` value for the `Skip higher-order stream used by the
+    /// `TimelineSubscriber`. See its documentation to learn more.
```

Comment:
Can you add brackets around `TimelineSubscriber` so this links to it?

Reply:
It's a private type and intra-links are disabled for private types, so it won't work (I've tried and got crazy about this 😋).